### ISSUE TITLE
Swagger UI contains link having incorrect URL (related to obtaining an access token)

### ISSUE THREAD

**Issue by eecavanna** [microbiomedata/nmdc-server#1361]
I don't know what the author's intention was—in other words, I don't know what information he or she intended to link to.

Screenshot of link (in dev environment): https://data-dev.microbiomedata.org/api/docs

![image](https://github.com/user-attachments/assets/a6c69189-97df-4cdc-a0b2-ca717092a41e)

When I click on that link, I end up at this page: https://data-dev.microbiomedata.org/user

![image](https://github.com/user-attachments/assets/67544c78-ebac-4eb1-9b36-c662d5f1c266)

In other words, I "must log in" to view the page where I went to learn how to log in. 🐔 🥚

**Comment by pkalita-lbl** [comment]
Haha yes that was the author's (a.k.a. my) intention 😁 

If you see the "You must log in to view this page" message at https://data-dev.microbiomedata.org/user I would expect you to log into the portal in the usual way (the "ORCID Login" button in the header). Then you should be able to access https://data-dev.microbiomedata.org/user and get the necessary token.

Perhaps the "You must log in..." message should contain a link to initiate the portal login process?

**Comment by eecavanna** [comment]
Thank you! Once I was logged into the Data Portal via the "ORCID Login" button, I was able to access that `/user` page.

<img width="809" alt="image" src="https://github.com/user-attachments/assets/282af74d-84e6-486b-a65d-fd57fae72472">

(Then, by following the instructions on that `/user` page, I was able to fetch my user information from `/api/me`.)

---

> Perhaps the "You must log in..." message should contain a link to initiate the portal login process?

I think so. I'll close this ticket and open a new one about doing that.

**Comment by eecavanna** [comment]
Here's a link to the newly-created issue (unassigned): https://github.com/microbiomedata/nmdc-server/issues/1362
